odbc/src/api/driver_connect_tests.rs

```diff
@@ -1,142 +0,0 @@
-macro_rules! test_connection_diagnostics {
```

Comment:
This file was initially deleted in SQL-1036 and was accidentally recreated after a git merge.  Deleting it again here.